src/Services/src/Tanzu.Toolkit.Services.csproj

```diff
@@ -1,23 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
Most of the changes in this file look to be related to reformatting.  Ideally reformatting changes are done separately from logic changes.  Reformatting changes should be isolated into their own changes sets.

Reply:
Fair, though it's a bit of a pain to check out a new branch just for some indentations in the middle of a hot fix 😄 
Feels like it's past time to get an effective linter enforcing consistent spacing so we don't need to reformat so often in the first place. We have a StyleCop ruleset but I think it's mostly just giving warnings at the moment -- doesn't feel like it's actually preventing us from pushing inconsistent code styles... Would an `.editorconfig` take care of this?

Reply:
`.editorconfig` would ensure consistency for tabs vs spaces and width, and can be set per file type, thought it is applied by your IDE (not sure if there's a CI check available for validation). I don't think Stylecop would work here, but in general we can make the build fail on warnings and that would force Stylecop validations to be addressed